src/app.hooks.js

```diff
@@ -118,6 +119,30 @@ const displayInternRequests = (level) => (context) => {
 	return context;
 };
 
+/**
+ * For errors without error code create GeneralError with code 500.
+ * @param {context} context
+ */
+const errorHandler = (context) => {
+	if (context.error) {
+
+		// to much for logging...
+		if (context.error.hook) {
+			delete context.error.hook; 
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Trailing spaces not allowed. (no-trailing-spaces)](https://app.codacy.com/app/schul-cloud/schulcloud-server/pullRequest?prid=4527512)